docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -14,6 +14,8 @@ This document describes how to prepare a Spryker project for going live on Spryk
 
 We divided the preparation into approximate timeframes, and you can adjust them to your needs. Make sure that all the following tasks are completed one week before going live.
 
+## Eight weeks before go-live
```

Comment:
The same header after each other doesnt seem to make sense

Reply:
Please let me know if my suggestions communicate the preparation timeframes accurately